.env

```diff
@@ -1,4 +1,4 @@
-RUBY_VERSION=3.2.2
+RUBY_VERSION=3.2.1
```

Comment:
@kev1njodea @michaelhagedon Is the Ruby version change still needed with the other changed made on main?